### PR TITLE
Add performance results processor script, update tests and codeowners

### PR DESCRIPTION
**Pull request by VladimirWork**

### CODE REVIEW

File: system/perf_res_processor.py [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+                                ['egrep', 'ERROR', path + log_name]
+                            ).decode().strip().splitlines()
+                        except subprocess.CalledProcessError:
+                            res += []
+
+                        # grep patterns and add them too
+                        for item in LOG_PATTERNS:
+                            try:
+                                pattern_res += subprocess.check_output(
+                                    ['egrep', item, path + log_name]
```
Comment (anikitinDSR) [inline system/perf_res_processor.py]: i think, that xzgrep it's enough for finding

File: system/perf_res_processor.py [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+        )
+
+        summary_file_names = sorted(
+                [x for x in os.listdir(path_from) if x.__contains__('summary')],
+                key=NATURAL_SORTING
+            )
+
+        # copy metrics summary for each node
+        assert all(
+            [copyfile(path_from+summary_file_name, path_to+summary_file_name) is not None
```
Comment (anikitinDSR) [inline system/perf_res_processor.py]: i think, that os.path.join is more clearly

File: system/perf_res_plotter.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        'max_node_prod_time',
+        'timestamp'
+    ]
+    for path, title in zip(paths, titles):
+        try:
+            pd.read_csv(path).loc[:, metrics].plot(
+                x='timestamp', subplots=True, cmap='cool', title=title, figsize=(20, 10),  # logy=True
+            )
+        except pd.errors.EmptyDataError:
+            plt.plot()
```
[the remark this answers is not in the document]
Comment (VladimirWork) [reply system/perf_res_plotter.py]: Added plt.clf() for clean plot.

File: system/perf_res_processor.py [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+import datetime
+import pandas as pd
+from pandas.plotting import table
+import matplotlib.pyplot as plt
+from system.perf_res_plotter import plot_metrics
+
+
+NODES_NUM = 25
+BASE_DIR = '/home/indy/indy-node/scripts/ansible/logs/'
+NODE_INFO_DIR = BASE_DIR + 'node_info/'
```
[the remark this answers is not in the document]
Comment (VladimirWork) [reply system/perf_res_processor.py]: Done with class with static attributes.

File: system/perf_res_processor.py (hunk 1 above)
Comment (VladimirWork) [reply system/perf_res_processor.py]: Done.

File: system/perf_res_processor.py (hunk 2 above)
Comment (VladimirWork) [reply system/perf_res_processor.py]: Done.

File: system/perf_res_processor.py [hunk 5]
```diff
@@ -32,6 +26,17 @@
 ]
 
 
+class PathReg:
+    base_dir = '/home/indy/indy-node/scripts/ansible/logs/'
```
Comment (anikitinDSR) [inline system/perf_res_processor.py]: I think, that base_dir as property will be more clearly. Like:
```
class PathReg:
   def __init__(self, base_dir='/home/indy/indy-node/scripts/ansible/logs/'):
      self._base_dir=base_dir

   @property
   def base_dir():
      return self._base_dir

   @property
   def node_info_dir():
      return os.path.join(base_dir, 'node_info')

   @property
   def jctl_dir():
       return os.path.join(base_dir, 'jctl')
```

.....
etc

Comment (VladimirWork) [reply system/perf_res_processor.py]: Done.

**Pull request merged**